packages/functions/src/public-types.ts

```diff
@@ -72,6 +72,30 @@ export interface Functions {
   customDomain: string | null;
 }
 
+/**
+ * Functions error code string before adding "functions/" product prefix.
```

Comment:
Staging it won't add anything that isn't here, unfortunately. Users were reporting that in v8, error codes were coming back as, for example, "deadline-exceeded" (one of the possible strings here) and in v9 they are coming back (more correctly) as "functions/deadline-exceeded" (since this is the functions package). We try to put a prefix with the SDK package name and a "/" before every error thrown by that SDK package. So Firestore should be throwing errors with code "firestore/something", auth should be throwing errors with code "auth/something", etc.

Reply:
Thanks!

So yes, if this is how it works, I'd expect something more like "* Functions error code string following the "functions/" product prefix."  

or maybe "* Functions error code string appended after "functions/" product prefix."

WDYT?

Reply:
Oh, I see. I've changed it to "appended after".